### ISSUE TITLE
Archive this repository

### ISSUE THREAD

**Issue by matozoid** [javaparser/javasymbolsolver#368]
When enough stuff has been moved to the javaparser repo.

**Comment by ftomassetti** [comment]
Not sure if we should keep it just to point to to JP. We could also renamed it to `javasymbolsolver_MOVED` (GitHub add redirects for renamed projects)

**Comment by un0btanium** [comment]
I am currently using older versions of JavaParser and JavaSymbolSolver. If i want to migrate to the new merged version, which maven dependency do i need to install.

These two together seem to work for now:
```
		<dependency>
		    <groupId>com.github.javaparser</groupId>
		    <artifactId>javaparser-core</artifactId>
		    <version>3.5.11</version>
		</dependency>
		
		<dependency>
		    <groupId>com.github.javaparser</groupId>
		    <artifactId>javaparser-symbol-solver-core</artifactId>
		    <version>3.5.11</version>
		</dependency>
```

JavaParser-Core does not come with the Symbol Solver yet, does it? When is it expected to come in one dependency?

By the way, putting these projects together is a great step forward! Looking forward to all the upcomming changes.

**Comment by ftomassetti** [comment]
Yes, from now on just put the same version for both the modules and you are done.

I think you can also just include the javaparser-symbol-solver-core and it will depend on javaparser-core, so it will be included automatically. At least, I would expect it to work in this way

**Comment by un0btanium** [comment]
You are right. Can confirm that only including `javaparser-symbol-solver-core` works like a charm!

Just read a few minutes ago on Twitter that the symbol solver has been put into the java parser repo, but only including the javaparser-core dependency in my project didnt include the symbol solver part. Is this going to be the long term plan, because I think most people arrive at the JavaParser repo and use the Maven dependency text from there, not realizing that the Symbol Solver doesnt come with it.

**Comment by ftomassetti** [comment]
Does, it is not the plan. We will have several modules under the JavaParser repository. Some people will be happy using just the javaparser-core module while others will need javaparser-symbol-solver-core. They will need to figure out they need to add a separate module. We will try to make it clear (and we are open to suggestions on this) but we expect the users to read the Readme (that we should update)

**Comment by un0btanium** [comment]
I see, that makes sense too.

I think in general having a nice looking and well structured readme is always important, not only for providing useful information, but also for having a professional appearances for new users.

**Comment by ftomassetti** [comment]
I think it is now possible to archive projects so I will rename this issue

**Comment by matozoid** [comment]
I've transfered the remaining issues, so here we go!
